recipes/munge_install.rb

```diff
@@ -50,9 +50,7 @@
     make -j $CORES
     make install
   MUNGE
-  # TODO: Fix, so it works for upgrade
-  creates '/usr/bin/munge'
-  not_if "/usr/sbin/munged --version | grep -q munge-#{node['cfncluster']['munge']['munge_version']}"
+  not_if "/usr/sbin/munged --version | grep -q munge-#{node['cfncluster']['munge']['munge_version']} && ls #{munge_libdir}/libmunge*"
```

Comment:
what happens if an unexpected version of munge is installed in `munge_libdir`?

Reply:
If unexpected means same version but not installed by us it should be ok.
If unexpected means different version, then the chef resource is triggered and the libraries are overridden